src/api/deposit/batchExchangeAddresses.ts

```diff
@@ -3,9 +3,9 @@ import { Network } from 'types'
 import { getGpV1ContractAddress } from 'utils/contract'
 
 const addressesByNetwork = {
-  [Network.Rinkeby]: getGpV1ContractAddress(Network.Rinkeby) as string,
-  [Network.Mainnet]: getGpV1ContractAddress(Network.Mainnet) as string,
-  [Network.xDAI]: getGpV1ContractAddress(Network.xDAI) as string,
+  [Network.Rinkeby]: getGpV1ContractAddress(Network.Rinkeby) || '',
```

Comment:
should the function just return empty string? anyways i think you know what you're doing here more than me so it's probably fine